.travis.yml

```diff
@@ -3,49 +3,12 @@ language: php
 matrix:
   include:
     - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7.0
-      env:
-        - DEPS=lowest
     - php: 7.0
-      env:
-        - DEPS=latest
     - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=latest
-    - php: 7.2
-      env:
-        - DEPS=lowest
     - php: 7.2
-      env:
-        - DEPS=latest
     - php: 7.3
-      env:
-        - DEPS=lowest
-    - php: 7.3
-      env:
-        - DEPS=latest
-    - php: 7.4
-      env:
-        - DEPS=lowest
     - php: 7.4
-      env:
-        - DEPS=latest
-    - php: nightly
-      env:
-        - DEPS=lowest
-        - COMPOSER_FLAGS="--ignore-platform-reqs"
     - php: nightly
```

Comment:
This should be changed to `8.0`.

Reply:
Thanks I do not follow closely what's going on in the Travis CI world these days, I was not aware the 8.0 was available.